Kernel/RandomValuePool.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+namespace Kernel {
+
+template<typename ValueType, size_t PoolSize>
+class RandomValuePool {
```

Comment:
o_O

Reply:
Oh that wasn't supposed to be committed ;-) That was meant to be for some work I started on for #4912... Damn `git commit -a`